.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
Even if I totally understand your thought about the difference between such dependency and whatever package you'll directly use in your code